src/plugins/discover/public/application/main/components/sidebar/discover_index_pattern_management.test.tsx

```diff
@@ -20,7 +20,7 @@ const mockServices = {
     },
   }),
   capabilities: {
-    visualize: {
```

Comment:
AFAIK capabilities are not coupled to the plugin id - I would like to keep it as `visualize` if possible, I think this will cause a bunch of downstream problems.